### PR TITLE
Add weekly update 2016-04-23

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

**Comment by Knighton910** [comment]
LGTM  with nits addressed

File: locale/en/blog/weekly-updates/weekly-update.2016-04-23.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### New Node.js Foundation Survey Reports New “Full Stack” In Demand Among Enterprise Developers
+
+The Node.js User Survey report features insights on emerging trends happening in this massive community that serves as a leading indicator on trends like microservices architectures, real-time web applications, Internet of Things (IoT). The report paints a detailed picture of the technologies that are being used, in particular, with Node.js in production and language preferences (current and future) for front end, back end and IoT developers.
+
+See https://nodejs.org/en/blog/announcements/nodejs-foundation-survey/ for more information.
+
+### Community Updates
+
+* [NodeUp podcast episode 100](http://nodeup.com/onehundred): A Creative Coding Show with Mariko Kosaka, Kassandra Perch, Myles Borins, and Rich Trott.
```
Comment (ghost) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-23.md:17]: didn't we have this last week?

Comment (Knighton910) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-23.md:17]: no, but we did have the full stack thing. Good eye though.

File: locale/en/blog/weekly-updates/weekly-update.2016-04-23.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+author: Minwoo Jung (github.com/JungMinu)
+date: 2016-04-23T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-04-23
+layout: blog-post.hbs
+---
+
+### Node.js News
+Node v5.11.0 (Stable) is released.
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-23.md]: I'd remove lines 11 and 12 as the only news item is repeated prominently in line 14.

File: locale/en/blog/weekly-updates/weekly-update.2016-04-23.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### Node v5.11.0 (Stable) Release
+
+We have one release: [Node v5.11.0 (Stable)](https://nodejs.org/en/blog/release/v5.11.0/). Complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-04-23.md]: Maybe remove `We have one release:`? Probably no need to mention the number of releases unless there is more than one.

**Comment by Knighton910** [comment]
So far 12 **Apr**   Are we adding this again? 

screen shot 2016-04-22 at 9 17 02 am

**Comment by JungMinu** [comment]
all nits addressed and updated :)

**Comment by Knighton910** [comment]
👍

**Pull request merged**